specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchindex.json

```diff
@@ -1479,7 +1479,7 @@
         "extractive"
       ],
       "x-ms-enum": {
-        "name": "Answers",
+        "name": "QueryAnswerType",
```

Comment:
I thought this was proposed as only an SDK change, this change is breaking and needs services changes to support it.

Reply:
Enum names don't appear on the wire anywhere AFAIK; This should be a client-only change.

Reply:
That is right, I misread which part was changing here. From a consistency standpoint should enum type names also change such as Captions and Speller? It seems odd to just change this one.

Reply:
That is a good suggestion. I updated those enum names too. I checked and they were not in the latest GA.